system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs

```diff
@@ -175,9 +175,16 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 2;
 }
 
+pub struct VestedTransferCalls;
+impl Contains<RuntimeCall> for VestedTransferCalls {
+	fn contains(call: &RuntimeCall) -> bool {
+		matches!(call, RuntimeCall::vesting(VestingCall::vested_transfer { .. }))
+	}
+}
+
 // Configure FRAME pallets to include in runtime.
 impl frame_system::Config for Runtime {
-	type BaseCallFilter = frame_support::traits::Everything;
+	type BaseCallFilter = frame_support::traits::EverythingExcluding<VestedTransferCalls>;
```

Comment:
```suggestion
	type BaseCallFilter = frame_support::traits::EverythingBut<VestedTransferCalls>;
```

Reply:
Yea sorry, wrote this from memory 😅

Reply:
I even searched traits thinking it was weird but maybe there's an alias 😅